.clang-tidy

```diff
@@ -12,5 +12,6 @@ Checks: 'clang-diagnostic-*,
 	,-readability-else-after-return,
 	,-readability-magic-numbers,
 	,-readability-function-cognitive-complexity,
+	,-readability-suspicious-call-argument,
 	,modernize-*,
```

Comment:
This flag triggers a lot when functions have parameters that are just a bunch of ints. For a c-language codebase, it's basically useless.